src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -472,6 +475,50 @@ void GeneralPage::initLayout(SettingsLayout &layout)
             return fuzzyToInt(args.value, 0);
         });
 
+    layout.addTitle("Streamer Mode");
+    layout.addDescription(
+        "Chatterino can automatically change behavior if it "
+        "detects that \"OBS Studio\" is running.\nSelect which "
+        "things you want to change while streaming");
+
+    layout.addDropdown<std::underlying_type<StreamerModeSetting>::type>(
+        "Enable Streamer Mode", {"Yes", "No", "Detect OBS (win)"},
+        s.enableStreamerMode,
+        [](int val) {
+            switch (val)
+            {
+                case StreamerModeSetting::Enabled:
+                    return "Yes";
+                case StreamerModeSetting::Disabled:
+                    return "No";
+                case StreamerModeSetting::DetectObs:
+                    return "Detect OBS (win)";
+                default:
+                    return "";
+            }
+        },
+        [](DropdownArgs args) {
+            if (args.value == "Yes")
+            {
+                return StreamerModeSetting::Enabled;
+            }
+            else if (args.value == "No")
+            {
+                return StreamerModeSetting::Disabled;
+            }
+            else
+            {
+                // default to Detecting if OBS.exe is running
```

Comment:
I don't understand this comment. There probably shouldn't even be a comment here.

Reply:
default to, means it's actively defaulting to something. Wording is different from default is

Reply:
I tried following this code here:
https://github.com/Chatterino/chatterino2/blob/b67e20d9620c415229bebc190f08d70b13e0cf31/src/widgets/settingspages/GeneralPage.cpp#L321-L324